pkg/cri/store/image/image.go

```diff
@@ -48,6 +55,8 @@ type Image struct {
 	ImageSpec imagespec.Image
 	// Pinned image to prevent it from garbage collection
 	Pinned bool
+	// matchcomparer for each runtime class. For more info, see CriService struct
+	platformMatcherMap map[string]platforms.MatchComparer
```

Comment:
check case.. 

I've not looked it over well enough, do we need the map here and in the criService parent struct.